tests/infrastructure/utils.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from ocp_resources.deployment import Deployment
+from ocp_resources.kubelet_config import KubeletConfig
+
+LOGGER = logging.getLogger(__name__)
+
+
+def verify_tekton_operator_installed(client):
+    """Verify Tekton operator is installed and available."""
+    LOGGER.info("Verifying Tekton operator is installed and available")
+    failures = []
+    tekton_deployment = Deployment(
+        name="openshift-pipelines-operator",
+        namespace="openshift-operators",
+        client=client,
+    )
+    if not tekton_deployment.exists or tekton_deployment.instance.status.readyReplicas == 0:
```

Comment:
`readyReplicas == 0` what's the desired amount of replicas here?
maybe better to compare to it and not just check that they are > 0?